### ISSUE TITLE
Configure arrow-spacing to specify exactly one space.

### ISSUE THREAD

**Issue by leebyron** [eslint/eslint#3278]
It would be awesome if eslint could help us ensure that >1 space is marked as a lint failure while exactly one is not.

I anticipate only needing this for the "before" condition, to allow for line breaks after =>

```js
x   => x // fail
x => x // pass
x =>
  x // pass
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by michaelficarra** [comment]
I don't think your examples are rendering the way you intended.

**Comment by leebyron** [comment]
Ugh, Markdown! Thanks for letting me know

**Comment by mysticatea** [comment]
There are reasons that `no-multi-spaces` is not used?

**Comment by nzakas** [comment]
@leebyron here @mysticatea is correct, we have a rule that bans multiple spaces in a row. Does that solve your use case?

**Comment by gyandeeps** [comment]
@leebyron Waiting for you response

**Comment by ilyavolodin** [comment]
No response, so closing as this functionality already exists in another rule. Feel free to reopen if you have additional information.
